### DISCUSSION TITLE
Update Multiple PHP Docker versions (8, 8.1, 7.4) separately

### DISCUSSION THREAD

**Discussion by robgordon89** [renovatebot/renovate#17812] [Request Help] [answered]
I am having an issue where I want to update the docker tags of all PHP versions we build for our images.

I have setup a simple test repo 

https://github.com/mailergroup/test-renovate 

Tags to update are here https://github.com/mailergroup/test-renovate/blob/5ab7329c862017856cccded2490cffe754cfa685/.github/workflows/build.yml#L15

The ideal outcome here would be that we have a PR for each Major+Minor

- 8.1.8-fpm -> 8.1.10-fpm
- 8.0.13-fpm -> 8.0.23-fpm
- 7.4.29-fpm -> 7.4.30-fpm

Currently, you will notice we only have one PR created for PHP 8.1 how can we group them by major-minor?

It will always currently update as below

- 8.1.8-fpm -> 8.1.10-fpm
- 8.0.13-fpm -> 8.1.10-fpm
- 7.4.29-fpm -> 8.1.10-fpm

That is why I turn of Major and Minor in the renovate config.

Thanks 
Bob

**Comment by viceice** [comment, marked as answer]
https://docs.renovatebot.com/configuration-options/#separateminorpatch

and disable or require approval for minor and major

**Reply by robgordon89** [reply]
how did i not see that option 😆 thanks
